Pipfile

```diff
@@ -6,9 +6,14 @@ name = "pypi"
 [packages]
 datadog = "*"
 black = "*"
+jsonschema = "*"
+requests = "*"
 
 [dev-packages]
 flake8 = "*"
 
 [requires]
 python_version = "3.7"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Oh I didn't notice black in the list of packages. My concern is that this would install prereleases of any of our packages. Typically I use pre-commit with Black, which involves adding this file to the repo:

```yml
repos:
-   repo: https://github.com/ambv/black
    rev: stable
    hooks:
    - id: black
language_version: python3
```

Then install pre-commit (I suggest `brew install pre-commit` for macOS) and run `pre-commit install`. This will then run black over modified files whenever you create a commit. You can also run it manually using `pre-commit run --all-files`. This way you don't need to include it in your Pipfile.

If you **do** include it in your Pipfile, it should really be a dev dependency.

Reply:
Thanks; I did land the travis-ci yaml stuff over in https://github.com/mozilla/wpt-api/commit/8753d489a397efe5ea8b2eeed0c41d8d2dacdcdb#diff-82c3b36709926de3a1b27bea0d8258cc, but haven't finished setting it up locally with ```pre-commit```.  Will look into this and fix ASAP.

Reply:
Hopefully fixed; removed black and configured locally.